src/resolve/resolved-dns-query.c

```diff
@@ -514,7 +515,7 @@ int dns_query_new(
                 .flags = flags,
                 .answer_dnssec_result = _DNSSEC_RESULT_INVALID,
                 .answer_protocol = _DNS_PROTOCOL_INVALID,
-                .answer_family = AF_UNSPEC,
```

Comment:
why was this removed?

Reply:
The comma was unintentionally removed when I removed a line that followed. I have re-introduced the comma here in the latest patch